### PR TITLE
update files for sidekick

### PR DESCRIPTION
**Pull request by louisachu**
update files for sidekick

## Description
update files for sidekick

## Related Issue
devsite-864

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by dmitrymatio — APPROVED** [review]
Needed for library plugin lgtm

**Review by dmitrymatio — CHANGES_REQUESTED** [review]
can this be merged into stage first?

**Comment by dmitrymatio** [comment]
> can this be merged into stage first?
Louisa: "it’s actually in stage already but I’m talking to the franklin library creator and he said these files need to be in main in order to see the library button, that’s why I’m just trying to merge these two files and leave what’s behind in stage."

**Review by dmitrymatio — APPROVED** [review]

File: tools/sidekick/config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "project": "adobe-io",
```
Comment (dylandepass) [inline tools/sidekick/config.json]: add `"host":  {your production host}"

as per https://www.hlx.live/docs/sidekick#customizing-the-sidekick

**Review by dylandepass — CHANGES_REQUESTED** [review]

File: tools/sidekick/library.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "project": "adobe-io",
+  "host": "https://www.hlx.live/docs/sidekick#customizing-the-sidekick",
```
Comment (dylandepass) [inline tools/sidekick/library.json:3]: You had it right before.. This needs to be your production hostname i.e developer.adobe.com?

**Review by dylandepass — CHANGES_REQUESTED** [review]

**Review by dylandepass — APPROVED** [review]

**Comment by dmitrymatio** [comment]
@dylandepass Thanks for the help!

**Comment by dylandepass** [comment]
@dmitrymatio Happy to help. Tell Dawn I say hi 👋🏻

**Pull request closed without merging**